dev/PushNotifications/PushNotificationUtility.cpp

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+#include "pch.h"
+#include "externs.h"
+#include "PushNotificationUtility.h"
+
+const std::wstring ConvertByteArrayToWideString(unsigned int payloadLength, _In_reads_(payloadLength) byte* payload)
+{
+    int size = MultiByteToWideChar(
+        CP_UTF8,
+        0,
+        reinterpret_cast<PCSTR>(payload),
+        payloadLength,
+        nullptr,
+        0);
+    THROW_LAST_ERROR_IF(size == 0);
+
+    std::wstring payloadAsWideString(size, 0);
+    size = MultiByteToWideChar(
+        CP_UTF8,
+        0,
+        reinterpret_cast<PCSTR>(payload),
+        payloadLength,
+        &payloadAsWideString[0],
+        size);
+    THROW_LAST_ERROR_IF(size == 0);
+
+    return payloadAsWideString;
+}
+
+void ProtocolLaunchHelper(unsigned int payloadLength, _In_reads_(payloadLength) byte* payload)
+{
+    // Command line format: ----WindowsAppRuntimePushServer:-Payload:"<payloadAsEscapedUriFormat>"
+    std::wstring commandLine = L"----WindowsAppRuntimePushServer:-Payload:\"";
```

Comment:
This is common code pasted from the LRP. Can we use a utility helper method here?